db/dohast.c

```diff
@@ -161,6 +161,10 @@ char *sqlite_struct_to_string(Vdbe *v, Select *p, Expr *extraRows,
         sqlite3_free(where);
         return NULL;
     }
+    if (!tbl && p->pSrc->nSrc) {
+        /* select 1 from tbl */
+        tbl = (const char*)p->pSrc->a[0].zName;
+    }
 
     if (unlikely(!tbl)) {
```

Comment:
Is this case still hit?  I guess it would be for cases like `select 1` but those won't be candidates for parallelization?

Reply:
Yes, the case is still hit, for example "select a from t1 union all select 1"